src/test/java/sync/google/GoogleKeywordDownloaderTest.java

```diff
@@ -91,4 +93,72 @@ public void testDownload() {
         assertThat(captor.getValue().campaignIds, containsInAnyOrder(1000));
     }
 
+    @Test
+    public void testDownloadTwoKeywordsVerifyTheFirst() {
```

Comment:
You successfully adopted 3 principles:
---------------------
1. test is short. made of only 3 statements
2. using builders and helper methods for 'english' reability
3. test scenario is using only local data and not a common setup

to improve
------------------------
1. test name: should say what is the situation and what is expected.
    for instance:

> test that BlaBla statue in google is mapped to Unknown status in kenshoo
        
> test that api call to google was made with same campaign ID in target given in the facade predicate

> test that id in google is mapped to inInTarget in kenshoo

2. don't test 2 keywords in 1 test

3. test every keyword field separately.

example
-----------------
```
void testThatIdInGoogleIsMappedToIdInTargetInKenshoo() {
    setupGoogleApi(
             withKeyword().withId(1).withCampaignId(1000)
    );

    KenshooKeyword downloadedKeyword = downloadOneKeyword(selectingCampaignId(1000));

    assertThat(downloadedKeyword.idInTarget, equalTo(1));
}
```

note that this way, I don't need to create a special matcher.
equalTo was enough.

Actually, it is my fault to request matchers in this exercise.